src/style.css

```diff
@@ -0,0 +1,267 @@
+/* stylelint-disable csstree/validator */
+
+/******** Reset *********/
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+:root {
+  --wine: rgb(159, 42, 61);
+  --soft-wine: rgba(159, 42, 62, 0.226);
+  --parrot: #e0989940;
+  --medium: #b4353527;
+}
+
+/******* Main section *********/
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100%;
+  flex-direction: column-reverse;
+  background-color: #d2c6c7be;
+  font-family: 'Raleway', sans-serif;
+}
+
+.logo {
+  position: absolute;
+  top: 24px;
+  left: 24px;
+  font-size: 1.5rem;
+  font-weight: 700;
+}
+
+.fa-circle-dot {
+  color: var(--wine);
+  font-size: 0.6rem;
+}
+
+.ocult {
+  display: none;
+}
+
+.checked {
+  text-decoration: line-through;
+  color: gray;
+}
+
+#title-container {
+  width: 100%;
+  height: 40px;
+  border-bottom: 1px solid var(--wine);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-family: 'Oswald', sans-serif;
+  color: var(--wine);
+  font-size: 1.3rem;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  background-color: var(--soft-wine);
+}
+
+.list-title {
+  margin-left: 12px;
+}
+
+.fa-arrows-rotate {
+  margin-right: 12px;
+  cursor: pointer;
+  color: var(--wine);
+}
+
+.fa-arrows-rotate:hover {
+  transform: scale(1.1);
+  color: var(--wine);
+}
+
+#task-form {
+  display: flex;
+  height: 40px;
+  width: 100%;
+  border: none;
+  justify-content: space-between;
+}
+
+.submit-button,
+#add-task {
+  border: none;
+  background-color: transparent;
+}
+
+#add-task {
+  padding-left: 12px;
+  width: 95%;
+}
+
+.submit-button {
+  cursor: pointer;
```

Comment:
Good Job!! 
To have your code more clean I recomend not repeat the classes and ID.